statsmodels/genmod/generalized_linear_model.py

```diff
@@ -208,17 +227,21 @@ def __init__(self, endog, exog, family=None, offset=None, exposure=None,
             exposure = np.log(exposure)
         if offset is not None:  # this should probably be done upstream
             offset = np.asarray(offset)
+
+        self.freq_weights = freq_weights
```

Comment:
why is this attached before super call?  If I remember correctly super attaches the extra arrays.
Otherwise, this would be the wrong array with dropped missing values.